package.json

```diff
@@ -283,7 +283,7 @@
     "node-abi": "^2.15.0",
     "node-addon-loader": "decred/node-addon-loader#master",
     "nouislider": "^12.0.0",
-    "pi-ui": "https://github.com/decred/pi-ui",
+    "pi-ui": "https://github.com/amassarwi/pi-ui#rbDescription",
```

Comment:
dev/test purposes, will revert and upgrade pi-ui when https://github.com/decred/pi-ui/pull/293 is merged.

Reply:
Can you revert this, so we can merge it?